rules/windows/lateral_movement_remote_services.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/11/16"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies remote execution of Windows services over Remote Procedure Call (RPC). This could be indicative of lateral
+movement but will be noisy if commonly done by admins."
```

Comment:
```suggestion
Identifies remote execution of Windows services over remote procedure call (RPC). This could be indicative of lateral
movement, but will be noisy if commonly done by administrators."
```